src/main/scala/cromwell/binding/CopyWorkflowOutputs.scala

```diff
@@ -0,0 +1,21 @@
+package cromwell.binding
+
+import cromwell.engine.WorkflowDescriptor
+
+import scala.concurrent.{ExecutionContext, Future}
+
+object CopyWorkflowOutputs {
+  val Name: LocallyQualifiedName = "$final_call$copy_workflow_outputs"
+}
+
+/**
+  * Final call implementation that copies workflow outputs to a specified destination.
+  */
+case class CopyWorkflowOutputs(workflow: WorkflowDescriptor) extends FinalCall {
```

Comment:
Is this really binding stuff? Binding is only stuff which is a generic representation of a WDL workflow and should minimize "what you do with it".